scalafmt-dynamic/src/main/scala/org/scalafmt/dynamic/ScalafmtDynamic.scala

```diff
@@ -51,7 +53,29 @@ final case class ScalafmtDynamic(
   override def withDefaultVersion(defaultVersion: String): Scalafmt = {
     copy(defaultVersion = defaultVersion)
   }
+
+  override def format(config: String, file: Path, code: String): String = {
+    val configFile = Paths.get("stdin.conf")
```

Comment:
This is just a dummy configuration path that will be passed to `ScalafmtReporter`.